CHANGELOG.md

```diff
@@ -4,6 +4,73 @@
+* [#7981](https://github.com/timescale/timescaledb/pull/7981) Add columnstore as alias for enable_columnstore in `ALTER TABLE`
+* [#7983](https://github.com/timescale/timescaledb/pull/7983) Support for SkipScan over compressed data
+* [#7991](https://github.com/timescale/timescaledb/pull/7991) Improves default segmentby options
+* [#7992](https://github.com/timescale/timescaledb/pull/7992) Add API into hypertable invalidation log
+* [#8000](https://github.com/timescale/timescaledb/pull/8000) Add primary dimension info to information schema
+* [#8005](https://github.com/timescale/timescaledb/pull/8005) Support `ALTER TABLE SET (timescaledb.chunk_time_interval='1 day')`
+* [#8012](https://github.com/timescale/timescaledb/pull/8012) Add event triggers support on chunk creation
+* [#8014](https://github.com/timescale/timescaledb/pull/8014) enable bool compression by default by setting `timescaledb.enable_bool_compression=true`. Note: for downgrading to `2.18` or earlier version one needs to use the provided downgrade script in 'timescaledb-extras'.
+* [#8018](https://github.com/timescale/timescaledb/pull/8018) Add spin-lock during recompression on unique constraints
+* [#8026](https://github.com/timescale/timescaledb/pull/8026) Allow `WHERE` conditions that use nonvolatile functions to be pushed down to the compressed scan level. For example, conditions like `time > now()`, where `time` is a columnstore orderby column, will evaluate `now()` and use the sparse index on `time` to filter out the entire compressed batches that cannot contain matching rows.
+* [#8027](https://github.com/timescale/timescaledb/pull/8027) Add materialization invalidations API
+* [#8047](https://github.com/timescale/timescaledb/pull/8027) Support SkipScan for `SELECT DISTINCT` with multiple distincts when all but one distinct is pinned
+
+**Bugfixes**
+* [#7862](https://github.com/timescale/timescaledb/pull/7862) Release cache pin when checking for `NOT NULL`
+* [#7909](https://github.com/timescale/timescaledb/pull/7909) Update compression stats when merging chunks
+* [#7928](https://github.com/timescale/timescaledb/pull/7928) Don't create a hypertable for implicitly published tables
+* [#7982](https://github.com/timescale/timescaledb/pull/7982) Fix crash in batch sort merge over eligible expressions
+* [#8008](https://github.com/timescale/timescaledb/pull/8008) Fix compression policy error message that shows number of successes
+* [#8031](https://github.com/timescale/timescaledb/pull/8031) Fix reporting of deleted tuples for direct batch delete
+* [#8033](https://github.com/timescale/timescaledb/pull/8033) Skip default segmentby if orderby is explicitly set
+* [#8061](https://github.com/timescale/timescaledb/pull/8061) Ensure settings for a compressed relation are found
+
+**GUCs**
+* `timescaledb.enable_bool_compression`
+* `timescaledb.enable_sparse_index_bloom`
```

Comment:
```suggestion
* `timescaledb.enable_sparse_index_bloom`
* `timescaledb.enable_event_triggers`
```

Reply:
thank you, added through https://github.com/timescale/timescaledb/pull/8068/commits/bdb1eca652c60088e93b3dc14c52096c961db203